nilearn/input_data/tests/test_nifti_labels_masker.py

```diff
@@ -125,6 +125,36 @@ def test_nifti_labels_masker_with_nans_and_infs():
     assert_true(np.all(np.isfinite(sig)))
 
 
+def test_nifti_labels_masker_reduction_strategies():
+    """Tests:
+           1. whether the usage of different reduction strategies work.
+           2. whether unrecognised strategies raise a ValueError
+           3. whether the default option is backwards compatible (calls "mean")
+    """
+    affine = np.eye(4)
+    img_data = np.array([[[-2., -1., 0., 1., 2],
+                          [-2., -1., 0., 1., 2]]])
+
+    labels_data = np.array([[[0, 0, 0, 0, 0],
+                             [1, 1, 1, 1, 1]]], dtype=np.int8)
+
+    img = nibabel.Nifti1Image(img_data, affine)
+    labels = nibabel.Nifti1Image(labels_data, affine)
+
+    strategies = ("mean", "median", "minimum", "maximum", "standard_deviation", "variance")
+    expected_results = (0, 0, -2, 2, np.sqrt(2), 2)
```

Comment:
Not a criticism, I have started using dicts for such situations.

Reply:
good suggestion. thanks!